gesture/gesture.go

```diff
@@ -37,6 +37,8 @@ type Click struct {
 	clicks int
 	// pressed tracks whether the pointer is pressed.
```

Comment:
Should this change to something like

// pressed tracks whether one or more buttons from the Buttons set is pressed.

?